src/modules/commander/commander.cpp

```diff
@@ -1702,7 +1702,7 @@ int commander_thread_main(int argc, char *argv[])
 	/* initialize low priority thread */
 	pthread_attr_t commander_low_prio_attr;
 	pthread_attr_init(&commander_low_prio_attr);
-	pthread_attr_setstacksize(&commander_low_prio_attr, PX4_STACK_ADJUSTED(3000));
+	pthread_attr_setstacksize(&commander_low_prio_attr, PX4_STACK_ADJUSTED(5000));
```

Comment:
We can only really afford this once this runs in the even library. @bkueng when we've moved this to events, this PR can go in.